clientv3/integration/watch_test.go

```diff
@@ -36,9 +36,8 @@ type watcherTest func(*testing.T, *watchctx)
 
 type watchctx struct {
 	clus          *integration.ClusterV3
-	w             clientv3.Watcher
```

Comment:
does this and `kv` need to change? There's a bunch of renames that don't seem necessary; `Client` already implements the interface so it should be a subtype and can be directly assigned into it...